examples/sample-allOptions.xml

```diff
@@ -24,7 +24,7 @@
  
  You can find more information about configuration options here: https://github.com/kohsuke/winsw/blob/master/doc/xmlConfigFile.md
 -->
-<configuration>
+<service>
```

Comment:
IIRC does not really matter

Reply:
It's not required by the current implementation. But what about the schema #68?

Reply:
Fair point